tests/menu.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
+
+// We want to make sure that the hover has all children visible and is clickable
+test('can select hovermenu', async ({ page }) => {
+    await page.goto('/');
+
+    await page.getByLabel('Ecosystem').hover();
+    const elements = [
+        page.locator(".section-submenu > a:nth-child(1)"),
+        page.locator(".section-submenu > a:nth-child(2)"),
+        page.locator(".section-submenu > a:nth-child(3)"),
+        page.locator(".section-submenu > a:nth-child(4)"),
+        page.locator(".section-submenu > a:nth-child(5)"),
+        page.locator(".section-submenu > a:nth-child(6)"),
+    ]
```

Comment:
I'd be surprised if there wasn't a "cleaner" way to get all direct children links

Reply:
There was and I fixed it :)